font-test-data/test_data/ttx/embedded_bitmaps.ttx

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I feel like some commentary on how this was created and what's in it might be helpful? - at a glance it's not super clear what's here or what the significance of each glyph is.

Glyphnames that were explanatory would go a long way, for example "noto_emoji_style_cbdt" would be more informative than "numbersign"

Reply:
Good point. I’ll make them after the formats they cover.

Reply:
Done!